<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maps_widget</class>
 <widget class="QWidget" name="Maps_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_maps">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_maps{background:rgb(20,20,20)}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>600</width>
      <height>200</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_select">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:none;
	background:none;
}

QLabel{
	color:white;
}</string>
     </property>
     <widget class="QFrame" name="frame_map">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>14</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/icons_white/Resources/Icons/png-white/maps-and-flags.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>88</x>
         <y>16</y>
         <width>98</width>
         <height>46</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Map</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>80</height>
        </rect>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/styles/Resources/appbox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="mapsSelectMapButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>80</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_6</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>mapsSelectMapButton</zorder>
     </widget>
     <widget class="QFrame" name="frame_directions">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>14</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/icons_white/Resources/Icons/png-white/direction.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>88</x>
         <y>16</y>
         <width>98</width>
         <height>46</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Directions</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>80</height>
        </rect>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/styles/Resources/appbox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="mapsSelectDirectionsButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>80</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_9</zorder>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>mapsSelectDirectionsButton</zorder>
     </widget>
     <widget class="QFrame" name="frame_home">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>14</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/icons_white/Resources/Icons/png-white/home.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>88</x>
         <y>16</y>
         <width>98</width>
         <height>46</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>80</height>
        </rect>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/styles/Resources/appbox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="mapsSelectHomeButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>80</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_21</zorder>
      <zorder>label_19</zorder>
      <zorder>label_20</zorder>
      <zorder>mapsSelectHomeButton</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="page_directions">
     <widget class="QFrame" name="frame_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>510</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background:rgba(57, 57, 57, 150);
	border-radius:5px;	

}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold Condensed</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	background:rgb(220, 0, 0);
	color:white;
	border-top-right-radius:0;
	border-bottom-right-radius:0;
}</string>
         </property>
         <property name="text">
          <string>From</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mapsDirectionFromInput">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	border:none;
	background:transparent;
	border-radius:5px;
	border-top-left-radius:0;
	border-bottom-left-radius:0;
}

QLineEdit:focus{
	background:rgba(57,57,57,150)
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>510</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background:rgba(57, 57, 57, 150);
	border-radius:5px;	

}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold Condensed</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	background:rgb(220, 0, 0);
	color:white;
	border-top-right-radius:0;
	border-bottom-right-radius:0;
}</string>
         </property>
         <property name="text">
          <string>To</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mapsDirectionToInput">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	border:none;
	background:transparent;
	border-radius:5px;
	border-top-left-radius:0;
	border-bottom-left-radius:0;
}

QLineEdit:focus{
	background:rgba(57,57,57,150)
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="mapsDirectionsHomeButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

	background:transparent;
	border:2px solid red;
	border-radius:25px;
}
QPushButton:pressed{
	background:red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons_white/Resources/Icons/png-white/home.png</normaloff>:/icons_white/Resources/Icons/png-white/home.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="mapsDirectionsGoButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

	background:transparent;
	border:2px solid red;
	border-radius:25px;
}
QPushButton:pressed{
	background:red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons_white/Resources/Icons/png-white/fi-rr-arrow-small-right.png</normaloff>:/icons_white/Resources/Icons/png-white/fi-rr-arrow-small-right.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_home">
     <widget class="QPushButton" name="mapsHomeGoButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

	background:transparent;
	border:2px solid red;
	border-radius:25px;
}
QPushButton:pressed{
	background:red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons_white/Resources/Icons/png-white/fi-rr-arrow-small-right.png</normaloff>:/icons_white/Resources/Icons/png-white/fi-rr-arrow-small-right.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QFrame" name="frame_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>510</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background:rgba(57, 57, 57, 150);
	border-radius:5px;	

}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold Condensed</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	background:rgb(220, 0, 0);
	color:white;
	border-top-right-radius:0;
	border-bottom-right-radius:0;
}</string>
         </property>
         <property name="text">
          <string>Ubication</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mapsHomeUbicationInput">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	border:none;
	background:transparent;
	border-radius:5px;
	border-top-left-radius:0;
	border-bottom-left-radius:0;
}

QLineEdit:focus{
	background:rgba(57,57,57,150)
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_title">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>600</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>380</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light</family>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>GOOGLE MAPS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>580</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:red;
border:none;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="mapsTitleCloseButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="icon">
      <iconset resource="../../resources.qrc">
       <normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-cross-small.png</normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-cross-small.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
